manual_system_tests/dae_scan.py

```diff
@@ -73,66 +67,37 @@ def dae_scan_plan() -> Generator[Msg, None, None]:
     controller.run_number.set_name("run number")
     reducer.intensity.set_name("normalized counts")
 
-    _, ax = yield from call_qt_aware(plt.subplots)
-
-    lf = LiveFit(
-        Linear.fit(), y=reducer.intensity.name, x=block.name, yerr=reducer.intensity_stddev.name
-    )
-
     yield from ensure_connected(block, dae, force_reconnect=True)
 
-    @subs_decorator(
-        [
-            HumanReadableFileCallback(
-                [
-                    block.name,
-                    controller.run_number.name,
-                    reducer.intensity.name,
-                    reducer.det_counts.name,
-                    dae.good_frames.name,
-                ],
-                output_dir=Path("C:\\")
-                / "instrument"
-                / "var"
-                / "logs"
-                / "bluesky"
-                / "output_files",
-            ),
-            LiveFitPlot(livefit=lf, ax=ax),
-            LivePlot(
-                y=reducer.intensity.name,
-                x=block.name,
-                marker="x",
-                linestyle="none",
-                ax=ax,
-                yerr=reducer.intensity_stddev.name,
-            ),
-            LiveTable(
-                [
-                    block.name,
-                    controller.run_number.name,
-                    reducer.intensity.name,
-                    reducer.intensity_stddev.name,
-                    reducer.det_counts.name,
-                    reducer.det_counts_stddev.name,
-                    dae.good_frames.name,
-                ]
-            ),
-            LiveFitLogger(
-                lf,
-                y=reducer.intensity.name,
-                x=block.name,
-                output_dir=Path("C:\\Instrument\\Var\\logs\\bluesky\\fitting"),
-                yerr=reducer.intensity_stddev.name,
-                postfix="manual_test",
-            ),
-        ]
+    icc = ISISCallbacks(
+        x=block.name,
+        y=reducer.intensity.name,
+        yerr=reducer.intensity_stddev.name,
+        fit=Linear.fit(),
+        measured_fields=[
+            block.name,
+            controller.run_number.name,
+            reducer.intensity.name,
+            reducer.intensity_stddev.name,
```

Comment:
I think we should assume that x, y, (y_err) if provided are measured - so this should be _extra_ measured fields in addition to those 2/3 I think